firmware/src/mPMTTestStand.cpp

```diff
@@ -224,7 +224,7 @@ void mPMTTestStand::execute()
     }
 
     // Check for any messages
-    if (this->comm.check_for_message()) {
+    if (this->comm.check_for_message() == SERIAL_OK) {
         uint8_t id = this->comm.received_message().id;
         DEBUG_PRINT_VAL("Received Message w/ ID", id);
         switch (id) {
```

Comment:
How does `handle_home_a()` relate to the axes. Similar for `handle_home_b()`?

Reply:
"Homing" has two steps:
- A: Move the axes in the negative direction until you hit the limit switch
- B: Move the axes in the positive direction until the limit switch releases

Reply:
Ah, that makes sense! Would it be possible to have a descriptions for this in future PR?

Reply:
Yeah sure, I will add them in the next next next PR cause already made the next and next next ones